### PR TITLE
Describe colors for which-key

### PR DESCRIPTION
**Pull request by elken**
## Description
Going to have a crack at finishing these off in the next few days. For now, this adds a simple line for the one colors.vim command.

Refs #17 

## Checklist
- [x] I have read
[CONTRIBUTING](https://github.com/MarcoIeni/intellimacs/blob/master/docs/CONTRIBUTING.org)
guidelines.

#### [CHANGELOG](https://github.com/MarcoIeni/intellimacs/blob/master/docs/CHANGELOG.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

#### [KEYBINDINGS](https://github.com/MarcoIeni/intellimacs/blob/master/docs/KEYBINDINGS.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

#### [PLUGINS](https://github.com/MarcoIeni/intellimacs/blob/master/docs/PLUGINS.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

### CODE REVIEW

File: spacemacs/colors.vim [hunk 1]
```diff
@@ -1,3 +1,4 @@
 " Pick color
+let g:WhichKeyDesc_Colors_ShowColorPicker = "<leader>Cl    ShowColorPicker"
```
Comment (marcoieni) [inline spacemacs/colors.vim]:
```suggestion
let g:WhichKeyDesc_Colors_ShowColorPicker = "<leader>Cl    colors-list"
```

Comment (marcoieni) [reply spacemacs/colors.vim]: Changed the name to something that corresponds to the keys and that follow the kebab-case convention.
What do you think?

Comment (elken) [reply spacemacs/colors.vim]: Et voila :)

**Comment by marcoieni** [comment]
Thanks!

**Pull request merged**